src/systems/ec/implementations/react/packages/footer/Footer.jsx

```diff
@@ -4,8 +4,28 @@ import classnames from 'classnames';
 
 import Link from '@ecl/ec-react-component-link/Link';
 
-const Footer = ({ identity, sections, common, className, ...props }) => (
-  <footer {...props} className={classnames(className, 'ecl-footer')}>
+const Footer = ({
+  backToTop,
+  identity,
+  sections,
+  common,
+  className,
+  ...props
+}) => (
+  <footer
+    {...props}
+    className={classnames(className, 'ecl-footer', {
+      'ecl-footer--custom': identity && identity.title,
+    })}
+  >
+    {/* Back to top */}
+    {backToTop && (
```

Comment:
The condition is not good enough. If the user provides `backToTop: {}`, then the condition is truthy (test in your console: `if ({}) console.log('the test passed')`, you'll see). 

A better check would look like `backToTop && backToTop.label && backToTop.label.length > 0`

Reply:
if think that `backToTop && backToTop.label` should be enough